### PR TITLE
Cliffh22

### PR DESCRIPTION
**Pull request by chull75**
Updated GUI - Not finished yet but I would love some feedback on what I have.

### CODE REVIEW

File: src/edu/psu/sweng500/team8/gui/BoardGUI.java [hunk 1]
```diff
@@ -23,6 +26,28 @@
...
+class BackgroundPanel extends JPanel
+{
+  Image image;
+  public BackgroundPanel()
+  {
+    try
+    {
+      image = javax.imageio.ImageIO.read(new java.net.URL(getClass().getResource("/blossom01.png"), "/blossom01.png"));
+    }
+    catch (Exception e) { /*handled in paintComponent()*/ }
```
Comment (jernelson7) [inline src/edu/psu/sweng500/team8/gui/BoardGUI.java]: doesn't look like paintcomponent handles anything

File: src/edu/psu/sweng500/team8/gui/BoardGUI.java [hunk 2]
```diff
@@ -33,15 +58,15 @@
...
 	public BoardGUI() {
 
 		this.setPreferredSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
 		this.setMaximumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
 		this.setMinimumSize(new Dimension(BOARD_SIZE, BOARD_SIZE));
 		this.setBorder(DEFAULT_BORDER);
-
-		this.blocks = new BlockGUI[3][3];
+				
+				this.blocks = new BlockGUI[3][3];
```
Comment (jernelson7) [inline src/edu/psu/sweng500/team8/gui/BoardGUI.java]: looks like tab formatting might be off

File: src/edu/psu/sweng500/team8/gui/SudokuGUI_Mark2.java [hunk 3]
```diff
@@ -51,6 +57,7 @@
 	 * Creates new form SudokuGUI
 	 */
 	public SudokuGUI_Mark2() {
+		setTitle("Lion Sudoku");
```
Comment (jernelson7) [inline src/edu/psu/sweng500/team8/gui/SudokuGUI_Mark2.java]: Lion??

**Comment by jernelson7** [comment]
This looks pretty good to me. One side note -- we should disable maximizing the form since it messes up the control positions

**Comment by chull75** [comment]
If anyone wants to change anything just let me know. I picked Lion Sudoku as really a placeholder. I you want we can keep it, or we can change it to something else and I can change the image

**Pull request merged**